status/status.go

```diff
@@ -0,0 +1,162 @@
+/*
+ *
+ * Copyright 2014 gRPC authors.
```

Comment:
The reason is that I want to not have them associated with gRPC but with Google, because people always tell me "why do you use gRPC status" when not using gRPC :)))

Reply:
That makes sense, so now I clarify the origin of these codes in doc.go.

Unfortunately, the generated code in googleapis does other undesirable things like coupling tightly to protobuf concepts and the protobuf runtime, so I really don't think it's a good idea to use that form verbatim.